Source/WTF/wtf/StringPrintStream.cpp

```diff
@@ -28,113 +28,107 @@
 
 #include <stdarg.h>
 #include <stdio.h>
-#include <wtf/FastMalloc.h>
-
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+#include <wtf/MallocSpan.h>
+#include <wtf/StdLibExtras.h>
 
 namespace WTF {
 
 StringPrintStream::StringPrintStream()
-    : m_buffer(m_inlineBuffer)
-    , m_next(0)
-    , m_size(sizeof(m_inlineBuffer))
 {
+    m_buffer = std::span<char> { m_inlineBuffer };
```

Comment:
Is the `<char>` needed?

Reply:
I'll try but usually yes. Because otherwise you end up with a `std::span<char, ARRAY_SIZE>` when constructing from an array.